core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1241,7 +1241,7 @@ public function showJsLibraryHint($dc)
 	public function addCteType($arrRow)
 	{
 		$key = $arrRow['invisible'] ? 'unpublished' : 'published';
-		$type = $GLOBALS['TL_LANG']['CTE'][$arrRow['type']][0] ?: '&nbsp;';
+		$type = ($GLOBALS['TL_LANG']['CTE'][$arrRow['type']][0] ?? null) ?: '&nbsp;';
```

Comment:
I agree with @aschempp 

```suggestion
		$type = ($GLOBALS['TL_LANG']['CTE'][$arrRow['type']][0] ?? $arrRow['type']) ?: '&nbsp;';
````
```

Reply:
`$type = $GLOBALS['TL_LANG']['CTE'][$arrRow['type']][0] ?? $arrRow['type'];` should be enough?

Reply:
True, yes. I wonder why it wasn't done like that before though?

Reply:
There was no fallback to `$arrRow['type']` previously, was there? Also, `$arrRow['type']` could theoretically be empty.

Reply:
> There was no fallback to $arrRow['type'] previously, was there?

correct, but it would make sense, and since this is only a label it shouldn't be a problem.

> Also, $arrRow['type'] could theoretically be empty.

Very unlikely, but yes, it would result in an empty string. You mean we should `??` on that property as well?

Reply:
I don't know, I was just trying to answer @fritzmg's question. 😄

Reply:
How could `$arrRow['type']` be empty/not in the array? 

So do we agree that we want to fall back to the type if not set?

Reply:
I don't think it will ever be empty. It _could_ be empty if you change the default type of your database and add an empty option. But the key would still be there, so I don't think we need to handle that case 😆

Reply:
adjusted in f97349c52